### PR TITLE
Fix some bugs, improve chat window

### PR DESCRIPTION
**Pull request by jaydenmilne**
- Fix destination card width in Firefox (#220)
- Fix routes not changing color after being claimed (#283), make it so entire path changes instead of just inside. This is based off feedback that it is too hard to tell which have been claimed
  Before:
  
  image

  After:
  
  image

- Chat Window Improvements
  - Increase font size
  - Decrease padding around message boxes
  - Messages and events associated with a player are now shown in that players color

    The server wasn't sending the required information
  - Make player name bold and colored, to stand out from the actual message
 
  - After:
    image

- Play soothing "bong" sound when it is your turn (https://soundbible.com/1424-Air-Plane-Ding.html)

### CODE REVIEW

File: client/src/app/game/turn/turn.service.ts [hunk 1]
```diff
@@ -67,6 +67,16 @@ export class TurnService {
     }
     else if (this.playerService.myPlayerId == this.playerService.activePlayerId) {
       this.playerNotifier.notifyPlayer('It is your turn!');
+
+      try {
+        let a = new Audio("assets/snd/yourturn.mp3");
```
Comment (gassajor000) [inline client/src/app/game/turn/turn.service.ts:72]: We have sounds now?

**Review by gassajor000 — APPROVED** [review]
looks good to me

File: client/src/app/game/turn/turn.service.ts (hunk 1 above)
Comment (jaydenmilne) [reply client/src/app/game/turn/turn.service.ts:72]: Yeah it beeps when its your turn

**Comment by jaydenmilne** [comment]
FYI changes are live at ticket2endgame.ga

**Pull request merged**